components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/internal/FrameworkServiceComponent.java

```diff
@@ -880,6 +911,22 @@ private void loadCodeForRequire() {
         }
     }
 
+    /**
+     * Load and read the JS function in require.js file.
+     */
+    private void loadCodeForSecrets() {
+
+        try {
+            ClassLoader loader = FrameworkServiceComponent.class.getClassLoader();
+            InputStream resourceStream = loader.getResourceAsStream("js/secrets.js");
+            secretsCode = IOUtils.toString(resourceStream);
+            FrameworkServiceDataHolder.getInstance().setCodeForSecretsFunction(secretsCode);
+        } catch (IOException e) {
+            log.error("Failed to read secrets.js file. Therefore, secrets object doesn't support in" +
```

Comment:
Better say "secrets object for secret resolution will not be available for adaptive auth scripts"

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/5723/commits/0b71da48f757b7651d5749723d7ed7b024393b6f